src/routes/routes.ts

```diff
@@ -17,6 +17,7 @@ export const Routes = {
     TRASH: '/trash',
     PROCESS_LOGS: `/process-logs/:id(${RESOURCE_UUID_PATTERN})`,
     SHARED_WITH_ME: '/shared-with-me',
+    RUN_PROCESS: '/run-process'
```

Comment:
we have `process-logs` so maybe we should rename `run-process` to `process-run` :)

Reply:
I prefer the first one: `run-process`